### ISSUE TITLE
What inherits from what? (StanDoc / ISO)

### ISSUE THREAD

**Issue by strogonoff** [riboseinc/metanorma#28]
Working on docs for Metanorma, and this is a passage currently appearing on metanorma.com in FAQ for StanDoc:

> Because of the richness of the ISO standards model, most Standoc standards to date (including the sample gem https://github.com/riboseinc/metanorma-sample) inherit from the ISO Standards model, which itself inherits from Standoc.

What purpose does it carry? Does it mean to suggest to the reader something along the lines of “Don’t use Standoc, use ISO, it has more features you will probably need”?

**Comment by opoudjis** [comment]
That paragraph is extremely important for programmers, and probably irrelevant for uses. The basic code is in standoc, and all standards classes customise that code. But you are not meant to use that code uncustomized, you are meant to always use a child gem. And that includes the Acme gem, in case you want to use default styling.

**Comment by opoudjis** [comment]
if you are writing an ISO standard, use the ISO gem. If you are you writing a Chinese standard comma use the GB gem. if a predefined gem is available for your standards class, that is what you are meant to be using. If one is not available, you should make one, or use the Acme gem as a skeleton.

**Comment by strogonoff** [comment]
I’m keeping this open until I’m sure metanorma.com reflects this well. I am relatively sure our primary visitors will be Metanorma end-users and we should move Standoc further under the hood.

**Comment by ronaldtse** [comment]
I agree with @strogonoff -- we should separate the site content per audience -- developers (whose organizations wants to adopt) vs users (whose organization already has an adoption). The latter is surely not interested in the content for the former, they only want to know how to use the syntax/toolchain relevant to their own documents.

**Comment by strogonoff** [comment]
As an update, the site now does separate the docs (/author and /builder). Builder documentation has ways to go, the plan is to document more low-level details, including the StanDoc inheritance chain.
